recipes/r-mapdata/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = '2.2-6' %}
+
+package:
+  name: r-mapdata
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: mapdata_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/mapdata_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/mapdata/mapdata_{{ version }}.tar.gz
+  sha256: dbaacb331ebd595898960ccf471f73491d9a4150ec1134e0954074b5bdd5704c
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  # Skipping Windows because r-maps is not available yet.
+  # See https://github.com/conda-forge/r-maps-feedstock/issues/1
+  skip: True  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-maps >=2.0_7
+    - posix  # [win]
+    - m2w64-toolchain  # [win64]
```

Comment:
The Windows `m2w64toolchain` is not the same thing. Note that all the r packages so far have this: https://github.com/conda-forge/r-maps-feedstock/blob/master/recipe/meta.yaml#L31 (only Windows), and then uses `gcc` on both `OS X` and `Linux`.